tox.ini

```diff
@@ -4,13 +4,13 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = {py34,py35,py36,pypy3}-{default,i386},flake8
+envlist = {py35,py36,py37,pypy3}-{default,i386},flake8
 
 [testenv]
 basepython = 
-    py34: python3.4
     py35: python3.5
     py36: python3.6
+    py37: python3.7
```

Comment:
This requires some changes to the .travis.yml as well: Python 3.7 needs to be installed using `pyenv`, and the `tox` call needs to be updated to include the `py37` environment.